linuxdeploy-plugin-gtk.sh

```diff
@@ -48,10 +62,58 @@ fi
 
 mkdir -p "$APPDIR"
 
-pushd "$APPDIR"
+echo "Installing AppRun hook"
+HOOKSDIR="$APPDIR/apprun-hooks"
+HOOKFILE="$HOOKSDIR/linuxdeploy-plugin-gtk.sh"
+mkdir -p "$HOOKSDIR"
+cat > "$HOOKFILE" <<\EOF
+#! /bin/bash"
+
+gsettings get org.gnome.desktop.interface gtk-theme 2> /dev/null | grep -qi "dark" && GTK_THEME_VARIANT="dark" || GTK_THEME_VARIANT="light"
+APPIMAGE_GTK_THEME=${APPIMAGE_GTK_THEME:-"Adwaita:$GTK_THEME_VARIANT"}
+CACHEDIR="$(mktemp --tmpdir --directory .AppRun.XXXXXXXX)"
```

Comment:
I did not find a way to clean this directory on exit. `trap "rm -rf $CACHEDIR" EXIT` does not work.

Reply:
Pretty sure the wrapper does `exec` the actual `AppRun`, so this cleanup can never be called. Maybe open an issue in linuxdeploy's tracker if you need this cleanup?

Reply:
Ok, linuxdeploy/linuxdeploy#133 opened.